### PR TITLE
docs: add release notes for 3.8.1 and 3.9.1 patch releases

### PR DESCRIPTION
**Pull request by kayx23**
Both released on April 29, 2024. (cc @Yilialinn )

### CODE REVIEW

**Review by AlinsRan — DISMISSED** [review]

File: blog/zh/blog/2024/04/29/release-apache-apisix-3.8.1.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+- Apache APISIX
+- API Gateway
+- API Management Platform
+- New Release
+- Cloud Native
+description: Apache APISIX 3.8.1 版本于 2024 年 4 月 29 日发布。该版本带来一则修复。
+tags: [Community]
+---
+
+我们很高兴发布 Apache APISIX 3.8.1。该版本带来一则修复以改善用户体验。
```
Comment (yzeng25) [inline blog/zh/blog/2024/04/29/release-apache-apisix-3.8.1.md]:
```suggestion
我们很高兴地宣布 Apache APISIX 3.8.1 版本已经发布。该版本带来一则修复以改善用户体验。
```

File: blog/zh/blog/2024/04/29/release-apache-apisix-3.9.1.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+- Apache APISIX
+- API Gateway
+- API Management Platform
+- New Release
+- Cloud Native
+description: Apache APISIX 3.9.1 版本于 2024 年 4 月 29 日发布。该版本带来一则修复。
+tags: [Community]
+---
+
+我们很高兴发布 Apache APISIX 3.9.1。该版本带来一则修复以改善用户体验。
```
Comment (yzeng25) [inline blog/zh/blog/2024/04/29/release-apache-apisix-3.9.1.md]:
```suggestion
我们很高兴地宣布 Apache APISIX 3.9.1 版本已经发布。该版本带来一则修复以改善用户体验。
```

**Review by yzeng25 — DISMISSED** [review]

File: blog/zh/blog/2024/04/29/release-apache-apisix-3.8.1.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+我们很高兴地宣布 Apache APISIX 3.8.1 版本已经发布。该版本带来一则修复以改善用户体验。
+
+<!--truncate-->
+
+## 修复
+
+### 修复 `forward-auth` 插件超时
+
+当客户端请求使用 POST 并且身份验证服务 API 需要 GET 时，修复 `forward-auth` 插件超时。该错误是由于 APISIX 将带有 `Content-Type` 和 `Expect` 等标头的 POST 请求转发到需要 GET 的身份验证服务 API 引起的。
```
Comment (Yilialinn) [inline blog/zh/blog/2024/04/29/release-apache-apisix-3.8.1.md]:
```suggestion
当客户端使用 POST 方法发送请求，但身份验证服务 API 实际上需要 GET 方法时，可能出现超时问题，为此我们针对 `forward-auth` 插件进行了修复。该问题源于 APISIX 在将带有`Content-Type` 和 `Expect` 等标头的 POST 请求转发给需要 GET 的身份验证服务 API 时出现的兼容性问题。
```

File: blog/zh/blog/2024/04/29/release-apache-apisix-3.8.1.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+tags: [Community]
+---
+
+我们很高兴地宣布 Apache APISIX 3.8.1 版本已经发布。该版本带来一则修复以改善用户体验。
+
+<!--truncate-->
+
+## 修复
+
+### 修复 `forward-auth` 插件超时
```
Comment (Yilialinn) [inline blog/zh/blog/2024/04/29/release-apache-apisix-3.8.1.md]:
```suggestion
### 修复 `forward-auth` 插件超时问题
```

File: blog/zh/blog/2024/04/29/release-apache-apisix-3.9.1.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+我们很高兴地宣布 Apache APISIX 3.9.1 版本已经发布。该版本带来一则修复以改善用户体验。
+
+<!--truncate-->
+
+## 修复
+
+### 修复 `forward-auth` 插件超时
+
+当客户端请求使用 POST 并且身份验证服务 API 需要 GET 时，修复 `forward-auth` 插件超时。该错误是由于 APISIX 将带有 `Content-Type` 和 `Expect` 等标头的 POST 请求转发到需要 GET 的身份验证服务 API 引起的。
```
Comment (Yilialinn) [inline blog/zh/blog/2024/04/29/release-apache-apisix-3.9.1.md]:
```suggestion
当客户端使用 POST 方法发送请求，但身份验证服务 API 实际上需要 GET 方法时，可能出现超时问题，为此我们针对 `forward-auth` 插件进行了修复。该问题源于 APISIX 在将带有`Content-Type` 和 `Expect` 等标头的 POST 请求转发给需要 GET 的身份验证服务 API 时出现的兼容性问题。
```

File: blog/zh/blog/2024/04/29/release-apache-apisix-3.9.1.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+tags: [Community]
+---
+
+我们很高兴地宣布 Apache APISIX 3.9.1 版本已经发布。该版本带来一则修复以改善用户体验。
+
+<!--truncate-->
+
+## 修复
+
+### 修复 `forward-auth` 插件超时
```
Comment (Yilialinn) [inline blog/zh/blog/2024/04/29/release-apache-apisix-3.9.1.md]:
```suggestion
### 修复 `forward-auth` 插件超时问题
```

**Comment by Yilialinn** [comment]
Please review @kayx23 @yzeng25 @AlinsRan

File: blog/en/blog/2024/04/29/release-apache-apisix-3.8.1.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
+---
+title: "Release Apache APISIX 3.8.1"
+authors:
+  - name: "Yuansheng Wang"
```
Comment (kayx23) [inline blog/en/blog/2024/04/29/release-apache-apisix-3.8.1.md:4]: Hi @AlinsRan - should this be you or? Who's working on the release this time

**Review by yzeng25 — APPROVED** [review]

**Review by moonming — APPROVED** [review]

**Review by Revolyssup — APPROVED** [review]

File: blog/en/blog/2024/04/29/release-apache-apisix-3.8.1.md (hunk 7 above)
Comment (AlinsRan) [reply blog/en/blog/2024/04/29/release-apache-apisix-3.8.1.md:4]: At first, it was him. I was just helping.

**Review by shreemaan-abhishek — APPROVED** [review]

**Review by AlinsRan — APPROVED** [review]

**Pull request merged**